### ISSUE TITLE
[Bug]: Unable to extend monster's languages

### ISSUE THREAD

**Issue by Gander7** [javalent/fantasy-statblocks#233]
### Check for existing bug reports before submitting.

- [X] I searched for existing [Bug Reports](https://github.com/javalent/fantasy-statblocks/issues?q=is%3Aopen+label%3Abug+sort%3Aupdated-desc) and found no similar reports.

### Expected Behavior

The Brass Dragon Wyrmling statblock has the language of Draconic.
The documentation states `Any field that is a plain string or an array can be added to, by specifying the name of the field followed by a +.`.

So I expected, when rendered, the languages to be `Draconic, Common, Infernal, Elvish`.

### Current behaviour

When rendered, the languages are still just Draconic.

### Reproduction

```statblock
monster: Brass Dragon Wyrmling
name: Dabshabah
languages+: Common, Infernal, Elvish
traits+:
  - name: Skilled Dancer
```

### Which Operating Systems are you using?

- [ ] Android
- [ ] iPhone/iPad
- [X] Linux
- [ ] macOS
- [ ] Windows

### Obsidian Version Check

1.3.5

### Plugin Version

3.6.5

### Confirmation

- [X] I have disabled all other plugins and the issue still persists.

### Possible solution

Wondering if this field was just missed when extensions were added as a feature.

**Comment by sigrunixia** [comment]
Hi @Gander7. This should be resolved in #249. Can you update the plugin and confirm it appears fixed for you?

**Comment by Gander7** [comment]
Good day @sigrunixia . Updated plugin and can confirm that it is fixed.
